src/item.cpp

```diff
@@ -2867,6 +2867,33 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
                     }
                 }
             }
+        } else if( is_gun() ) {
+
+            std::string bp_name;
+            bodypart_id bp_id;
+            for( const item *mod : gunmods() ) {
+                if( has_flag( flag_IS_ARMOR ) ) {
+                    //todo: add further checks for specific location flags whenever they get added
+                    if( mod->has_flag( flag_BELTED ) ) {
```

Comment:
The code blocks in `if` and in `else` are the same??

Reply:
it is redundant yeah ... i was mostly there to remind me to check for further location flags that gunmods might have. i ll remove it.